pyrene/src/components/MultiSelect/MultiSelect.examples.ts

```diff
@@ -46,25 +45,23 @@ const icons = ['place', 'layers', 'clock'];
 const colors = [colorConstants.blue600, colorConstants.red600, colorConstants.orange600, undefined];
 const testOptionsWithIcons = testOptions.map((option, i) => ({ ...option, iconProps: { name: icons[i % 3], color: colors[i % 4] } }));
 
-const makeExample = (options: Array<Option>) => ({
+const makeExample = (options: ReadonlyArray<Option>) => ({
```

Comment:
Oh that's cool, I did not know about `ReadonlyArray` yet 😄

Reply:
@cyrilwanner I needed to that way because of the dependency we are using, called `<Select />` .